src/layouts/Layout.js

```diff
@@ -6,15 +6,15 @@ import Logo from '../assets/images/protegeLogo.svg'
 import { motion } from 'framer-motion'
 
 const Layout = ({ children, variant }) => {
-  let variantClassName = 'py-2 absolute w-full md:px-3 z-50 bg-white shadow-md'
+  let variantClassName = 'py-2 absolute w-full md:px-8 z-50 bg-white shadow-md'
```

Comment:
Is this initial value necessary if the default ends up changing it in the switch case if `'home'` isn't the variant?

Reply:
@nickytonline You're correct. I'll pull this out. Thanks!